app/lib/fake_data/complaint.rb

```diff
@@ -0,0 +1,59 @@
+require "ffaker"
+
+module FakeData
+  class Complaint
+    def data
+      @data ||= {
```

Comment:
[praise] I like this mixing of memoization with using a new object on each loop of the generator. Feels like a really flexible approach moving forward.